plugins/com.google.cloud.tools.eclipse.appengine.localserver.test/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerLaunchConfigurationDelegateTest.java

```diff
@@ -186,8 +186,6 @@ public void testGenerateRunConfiguration() throws CoreException {
         .generateServerRunConfiguration(launchConfiguration, server);
     assertNull(config.getHost());
     assertEquals((Integer) LocalAppEngineServerBehaviour.DEFAULT_SERVER_PORT, config.getPort());
-    assertEquals(LocalAppEngineServerBehaviour.DEFAULT_ADMIN_HOST, config.getAdminHost());
-    assertEquals((Integer) LocalAppEngineServerBehaviour.DEFAULT_ADMIN_PORT, config.getAdminPort());
```

Comment:
The most likely outcome is:

1. devappserver2 will not be finished.
2. If it is finished, we still won't use it.
3. If it is finished and we use it, it will have changed enough that these tests are no longer accurate and/or relevant.